axes/migrations/0009_add_session_hash.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.2.2 on 2024-04-30 07:57
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("axes", "0008_accessfailurelog"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="accesslog",
+            name="session_hash",
+            field=models.CharField(
+                blank=True,
+                default="",
+                max_length=64,
```

Comment:
SHA256 should be exactly 64 characters so it might be sensible to force this to either None or 64 characters with e.g. [MinLengthValidator](https://docs.djangoproject.com/en/dev/ref/validators/#minlengthvalidator)